api/proxy/servers/rest/handlers_cert.go

```diff
@@ -56,11 +61,21 @@ func (svr *Server) handleGetOPKeccakCommitment(w http.ResponseWriter, r *http.Re
 
 // handleGetOPGenericCommitment handles the GET request for optimism generic commitments.
 func (svr *Server) handleGetOPGenericCommitment(w http.ResponseWriter, r *http.Request) error {
+	if !svr.config.EnabledAPIs.RestALTDAOPGeneric() {
+		w.WriteHeader(http.StatusForbidden)
+		return fmt.Errorf("op-generic DA Commitment type detected but `op-generic` API is not enabled")
+	}
```

Comment:
guessing you've considered turning these routes off altogether in router.go, but opted to keep them with more explicit error msg?